From 159b2c4d151b2655f98cc80bd17f59fa62c63303 Mon Sep 17 00:00:00 2001
From: Andy Yan <andy.yan@rock-chips.com>
Date: Thu, 7 May 2020 17:19:45 +0800
Subject: [PATCH] arm64: dts: rockchip: Remove data-sync property for
 px30-ad-d6-anx6345 rgb interface

The data-sync is enabled by default, so we don't need
this property.

Change-Id: I3cfbb051368a2065cf6db5d0bcc91b8e77e77d10
Signed-off-by: Andy Yan <andy.yan@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/px30-ad-d6-anx6345.dts | 1 -
 1 file changed, 1 deletion(-)

diff --git a/arch/arm64/boot/dts/rockchip/px30-ad-d6-anx6345.dts b/arch/arm64/boot/dts/rockchip/px30-ad-d6-anx6345.dts
index 0e7a2dcd8c0a..a51884b8de38 100644
--- a/arch/arm64/boot/dts/rockchip/px30-ad-d6-anx6345.dts
+++ b/arch/arm64/boot/dts/rockchip/px30-ad-d6-anx6345.dts
@@ -609,7 +609,6 @@
 	pinctrl-names = "default", "sleep";
 	pinctrl-0 = <&lcdc_rgb_pins>;
 	pinctrl-1 = <&lcdc_sleep_pins>;
-	rockchip,data-sync;
 	status = "okay";
 
 	ports {
-- 
2.35.3

